### DISCUSSION TITLE
Managed solution on the shelf

### DISCUSSION THREAD

**Discussion by romjeDecat** [arkmq-org/arkmq-org-broker-operator#669] [General] [answered]
Hi all, do you know if someone offer a fully managed (commercial with support) for Artemis, using artemis cloud solution ?
My customer would like to have such solution to reduce the pressure on the OPS teams..
Thanks for any pointer/feedback

Regards
Jerome

**Comment by gaohoward** [comment, marked as answer]
Hi @romjeDecat , Please contact Red Hat for commercial support. They have cloud solutions for Artemis (AMQ as its product name).
